python/pyspark/sql/functions.pyi

```diff
@@ -85,6 +85,9 @@ def lag(
 def lead(
     col: ColumnOrName, offset: int = ..., default: Optional[Any] = ...
 ) -> Column: ...
+def nth_value(
+    col: ColumnOrName, offset: int, ignoreNulls: Optional[bool] = ...
```

Comment:
I was about to suggest to make it accept `Column` in the original PR but .. let's leave it matched for now - I think Window once might have to be treated differently given that it has different SQL syntax.